scripts/CMakeLists.txt

```diff
@@ -52,9 +52,18 @@ if (INSTALL_SYSTEM_FILES)
 			DESTINATION ${BASH_COMPLETION_COMPLETIONSDIR}
 			RENAME kdb)
 
+	set (FISH_COMPLETION_COMPLETIONSDIR "/usr/share/fish/vendor_completions.d")
+	find_package (PkgConfig)
+	if (PKG_CONFIG_FOUND)
+		include (LibFindMacros)
+		pkg_get_variable (FISH_COMPLETION_COMPLETIONSDIR fish completionsdir)
```

Comment:
Please check first if the module fish exists to avoid the error:

```
Package fish was not found in the pkg-config search path.
Perhaps you should add the directory containing `fish.pc'
to the PKG_CONFIG_PATH environment variable
No package 'fish' found
CMake Error at scripts/CMakeLists.txt:61 (install):
  install FILES given no DESTINATION!
```